test-suite/src/test/scala/org/scalajs/testsuite/javalib/LinkedListTest.scala

```diff
@@ -0,0 +1,357 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___      __ ____  Scala.js Test Suite        **
+**    / __/ __// _ | / /  / _ | __ / // __/  (c) 2013-2015, LAMP/EPFL   **
+**  __\ \/ /__/ __ |/ /__/ __ |/_// /_\ \    http://scala-js.org/       **
+** /____/\___/_/ |_/____/_/ | |__/ /____/                               **
+**                          |/____/                                     **
+\*                                                                      */
+package org.scalajs.testsuite.javalib
+
+import scala.language.implicitConversions
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+
+import org.scalajs.jasminetest.JasmineTest
+
+import scala.scalajs.runtime.UndefinedBehaviorError
+
+import java.util.LinkedList
+import java.util.Comparator
+
+object LinkedListTest extends JasmineTest {
+
```

Comment:
You should not have test for everything here. You should reuse all the tests in `AbstractListTest` and only test the characteristics that are proper to `LikedList` in here. The `ArrayListTest` is a good example of that and you should mimic it. You should also create a `AbstractSequentialListTest` in between the two tests.